.github/workflows-source/ci-pr-only.yml

```diff
@@ -38,13 +37,6 @@ anchors:
       <<: *image
       options: >-
         -e NODE_NAME
-  before-script: &before-script
```

Comment:
I think it's beneficial if we get node info on all CI jobs that we run on on-prem runners, so I'd retain before-script in CI jobs of this workflow.